server/routes/v1/upload.ts

```diff
@@ -161,7 +165,7 @@ export const postUpload = [
       version.model = model._id
       await version.save()
 
-      req.log.info({ modelId: model._id }, 'Created model document')
+      req.log.info({ model: _.pick(model, ['_id', 'uuid', 'schemaRef']) }, 'Created model document')
```

Comment:
Not needed anymore with new serialisation method.

Reply:
Good spot, changed